### DISCUSSION TITLE
Set External Storage

### DISCUSSION THREAD

**Discussion by richardcrbb** [nextcloud/all-in-one#5048] [Questions] [answered]
Hi, I spend hours following this instructions:

¨You can then navigate to the apps management page, activate the external storage app¨
from [this](https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-allow-the-nextcloud-container-to-access-directories-on-the-host)

Cause it is not clear where to find this app, there is no search option, and the only apps that I found were 

-External Storage Backend for Dropbox
-External Storage Support for Google Drive
-External Storage: One Drive
-External Storage Swarm

I found the External storage apps in

Your Apps:

and it was disabled by default. you have to enable it.

As well I encountered a problem when running the docker cause the instructions said 

--env NEXTCLOUD_MOUNT="/mnt/"

But there is missing this           \           at the end.....

It should say something like 

--env NEXTCLOUD_MOUNT="/mnt/YOUR_DRIVE_MOUNTPOIN"  \

Cause without the \ at the end the docker will get a problem.

I know little about code and doing this following your instructions took me a lot of time. Please update this instructions!!!! People who don´t know code will appreciate it.

Thanks!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by szaimen** [comment, marked as answer]
Hi, 

> I found the External storage apps in
> 
> Your Apps:
> 
> and it was disabled by default. you have to enable it.

Thanks for this feedback. I improved the instructions via https://github.com/nextcloud/all-in-one/commit/562b55b872becbcb81bf262bd7e6e6f45db92e6b



> But there is missing this \ at the end.....
> 
> It should say something like
> 
> --env NEXTCLOUD_MOUNT="/mnt/YOUR_DRIVE_MOUNTPOIN" \
> 
> Cause without the \ at the end the docker will get a problem.
> 
> I know little about code and doing this following your instructions took me a lot of time. Please update this instructions!!!! People who don´t know code will appreciate it.

This doesnt has to do with coding. It is basic CLI knowledge. So I am not sure we should add this as it is different for example on windows (there you need to use `^`)....
